### PR TITLE
Updated the menu strucutre

### PR DESCRIPTION
**Pull request by linkfang**
Fixes https://github.com/openhwgroup/openhwgroup.org/issues/395

### CODE REVIEW

**Review by chrisguindon — CHANGES_REQUESTED** [review]
We don't want to delete the news items on this site. Many news items in the newsroom are configured to redirect back to the article on this website such as https://www.openhwgroup.org/news/2021/06/16/register-for-openhw-tv-s2-e06-core-v-extension-i/f-cvxif/

**Comment by linkfang** [comment]
> We don't want to delete the news items on this site. Many news items in the newsroom are configured to redirect back to the article on this website such as https://www.openhwgroup.org/news/2021/06/16/register-for-openhw-tv-s2-e06-core-v-extension-i/f-cvxif/

Sounds good!

I reverted the change.

File: layouts/partials/footer_links/suffix.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<script id="event-short-list" type="text/html">
```
Comment (chrisguindon) [inline layouts/partials/footer_links/suffix.html:1]: Do we need this on all pages? If not, I would recommend adding it to a different layout page such as https://github.com/openhwgroup/openhwgroup.org/blob/master/layouts/index.html

File: layouts/shortcodes/events.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<div class="col-sm-8 col-sm-offset-1 news-list">
```
Comment (chrisguindon) [inline layouts/shortcodes/events.html:1]: Are you using this new shortcode anywhere? If not, we should remove them.

**Review by chrisguindon — CHANGES_REQUESTED** [review]
I am noticing that we have a image folder under /content/news:

[content/news/images/](https://github.com/openhwgroup/openhwgroup.org/pull/397/files#diff-a2c87ca58c373b3fe77739bebb4375c3d70433756ca7905acbf050467591f740)


I would expect th

File: layouts/partials/footer.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!-- 
+<!--
```
[the remark this answers is not in the document]
Comment (linkfang) [reply layouts/partials/footer.html]: removed it

File: layouts/partials/footer_links/suffix.html (hunk 1 above)
Comment (linkfang) [reply layouts/partials/footer_links/suffix.html]: Moved it to layouts/index.html

File: layouts/shortcodes/events.html (hunk 2 above)
Comment (linkfang) [reply layouts/shortcodes/events.html]: Removed the shortcode

**Pull request merged**